apps/wallet-dashboard/components/VirtualList.tsx

```diff
@@ -31,6 +32,8 @@ export function VirtualList<T>({
     overflowClassName,
 }: VirtualListProps<T>): JSX.Element {
     const containerRef = useRef<HTMLDivElement | null>(null);
+    const lastElementRef = useRef<HTMLDivElement | null>(null);
+    const { isIntersecting } = useOnScreen(lastElementRef);
```

Comment:
Not needed, VirtualList already implements automatic fetching when the last item appears